### DISCUSSION TITLE
Explicitly remove -werror=implicit-interface flag.

### DISCUSSION THREAD

**Discussion by ajayrawat** [fortran-lang/fpm#1062] [General]
I'm using FPM in my project, which requires dispmodule. However, because of the -werror=implicit-interface flag, I'm not able to compile the code. I've used -flags to override the defaults, but still  "-werror=implicit-interface" exists.

**Comment by trueprecision** [comment]
YES!  This is a problem for me as well.   Quite annoying.  I don't understand why fpm doesn't respect  FPM_FFLAGS settings.  Back to using make...

**Reply by perazz** [reply]
https://github.com/fortran-lang/fpm/pull/864

```toml
[fortran]
implicit-external=true
```

**Reply by trueprecision** [reply]
Thank you.  I am now getting a clean compile.
